bucoffea/plot/util.py

```diff
@@ -207,6 +207,10 @@ def merge_datasets(histogram):
         'WNJetsToLNu-MLM_2017' : [x for x in all_datasets if re.match('W(\d+)JetsToLNu_2017',x)],
         'WNJetsToLNu-MLM_2018' : [x for x in all_datasets if re.match('W(\d+)JetsToLNu_2018',x)],
 
+        'Diboson_2017' : [x for x in all_datasets if re.match('(WW|WZ|ZZ|WW).*_2017',x)],
+        'Diboson_2018' : [x for x in all_datasets if re.match('(WW|WZ|ZZ|WW).*_2018',x)],
+        'EWKW2Jets_WToLNu_M-50-mg_2017' : [x for x in all_datasets if re.match('EWKW(Plus|Minus)2Jets.*-mg_2017',x)],
+        'EWKW2Jets_WToLNu_M-50-mg_2018' : [x for x in all_datasets if re.match('EWKW(Plus|Minus)2Jets.*-mg_2018',x)],
```

Comment:
Can this also be put into the yearly loop down below? Should be the default

Reply:
Sure, I can put that in the yearly loop too.

Reply:
Done in commit: https://github.com/alpakpinar/bucoffea/commit/dcf0dee0db3381b8cd0faa49bc3eded217a13a21